ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -30,12 +31,18 @@ function ScanConfigPage(): ReactElement {
     const { clusters, isLoading: isFetchingClusters } = useFetchClustersForPermissions([
         'Compliance',
     ]);
+    const [isCreating, setIsCreating] = useState(false);
 
     const listQuery = useCallback(() => listComplianceProfiles(), []);
     const { data: profiles, loading: isFetchingProfiles } = useRestQuery(listQuery);
 
-    function onCreate() {
-        // TODO: create scan
+    async function onCreate() {
+        setIsCreating(true);
+        const complianceScanConfig = convertFormikToScanConfig(formik.values);
+
+        await createScanConfig(complianceScanConfig);
```

Comment:
If the request that is kicked off here returns a non-2xx, axios will reject the Promise and cause `await createScanConfig(complianceScanConfig);` to throw, leaving `isCreating` in a permanent `true` state.

Ideally we would catch the error here, display it, and reset `isCreating` to `false`.

Reply:
Good callout, I added an `<Alert />` on failure